lib/notion/faraday/request.rb

```diff
@@ -27,6 +27,7 @@ def delete(path, options = {})
       def request(method, path, options)
         response = connection.send(method) do |request|
           request.headers['Authorization'] = "Bearer #{token}"
+          request.headers['Notion-Version'] = "2021-05-11"
```

Comment:
Could we create a `API_VERSION` constant in `version.rb` to store this info?

https://github.com/orbit-love/notion-ruby-client/blob/e6485456ae51ad1790cfb4a1bb9adef153a44dea/lib/notion/version.rb#L2-L4

Reply:
I’m not sure how frequently Notion is going to change their version number, but I think having it at a more prominent place in the codebase might help folks figure out which version we use

Reply:
Sure, that seems like a more maintainable solution.